system-addon/lib/ActivityStreamMessageChannel.jsm

```diff
@@ -138,7 +138,9 @@ this.ActivityStreamMessageChannel = class ActivityStreamMessageChannel {
     this.channel.addMessageListener("RemotePage:Load", this.onNewTabLoad);
     this.channel.addMessageListener("RemotePage:Unload", this.onNewTabUnload);
     this.channel.addMessageListener(this.incomingMessageName, this.onMessage);
+  }
 
+  dispatchInitLoadEvent() {
```

Comment:
Nice that this separated so easily. I would name this something else as there's already too many "dispatch" and "init"s. Probably something along the lines of `simulateNewTabEvents`.